java/runtime/src/main/java/io/ray/runtime/context/RuntimeContextImpl.java

```diff
@@ -34,6 +34,10 @@ public JobId getCurrentJobId() {
 
   @Override
   public ActorId getCurrentActorId() {
+    if (runtime.getRayConfig().runMode.equals(RunMode.SINGLE_PROCESS)) {
+      throw new UnsupportedOperationException("This API is not allowed in SINGLE_PROCESS mode.");
```

Comment:
```suggestion
      throw new UnsupportedOperationException("This API is not allowed in local mode.");
```

And I think it's better to move the check in `LocalModeWorkerContext`.